### PR TITLE
Finished desktop version

### PR DESCRIPTION
**Pull request by Boku52500**
- Added media in css to adjust sizings.
- Added new pictures to match the screen.
- Resized everything for desktop version.
- Fixed Cards.
- Fixed linters.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -445,3 +451,309 @@ h4 {
...
+
+.page-4-desktop {
+  display: none;
+}
+
+.page-5-desktop {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
```
Comment (Felix45) [inline style.css:573]: Please note that at screen widths greater than `1200px` your design is not consistent with the Figma template. The alignment of the project cards appears to be distorted. For screen sizes above `768px,` your layout needs to be responsive and consistent with the Figma template. Kindly see the screenshots below and amend this.

## Figma template





## Your Implementation at 1200px






## Your Implementation at 1400px

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @Boku52500 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

You've done well :clap: :clap: , just a few changes and your design will be good to go.

almost-there

## Highlights :heavy_check_mark: 
- [ ] Passing linter checks :clap: 
- [ ] Professional readme file


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Boku52500** [comment]
- Fixed responsive resolution for 1024px
- Fixed responsive resolution for 768px
- Fixed responsive resolution for 1440px
- Fixed linters

File: index.html [hunk 2]
```diff
@@ -12,283 +12,404 @@
...
-            </div>
-            <div class="alltext">
-                <ul>
-                    <li class="text1">html</li>
-                    <li class="text2">css</li>
-                    <li class="text3">javaScript</li>
-                </ul>
-            </div>
-            <button class="but">See Project</button>
-        </section>
```
Comment (sajid-munawar) [inline index.html:81]: - [ ] As the previous reviewer mention please follow figma template to match your design. kindly add an overall background color to your portfolio and give separate background color for every card and also give it a `border-radius`
## Your design👇
card
 ## Figma template👇
cardfigma

File: style.css [hunk 3]
```diff
@@ -445,3 +451,387 @@ h4 {
...
+
+  .right1 {
+    margin-left: 375px;
+  }
+}
+
+@media only screen and (min-width: 1440px) {
+  .but1 {
+    margin-top: -200px;
+  }
```
Comment (sajid-munawar) [inline style.css]: - [ ] Please remove the gap and make it like the figma template. after fixing this your card will look much more better.
## your design👇
getbtn

## Figma design 👇
getbtnFigma

**Review by sajid-munawar — CHANGES_REQUESTED** [review]
Hi @Boku52500 , 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Boku52500** [comment]
- Reconstructed the whole design from scratch
- Updated the designs for cards for all versions
- Fixed margins for all versions
- Fixed linters

**Review by mgmediaweb — APPROVED** [review]
Hi @Boku52500, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved5

## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**